<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 243, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="label_min">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(180, 180, 180);
	color: white;
	border: 1px solid gray;
}
QLineEdit: pressed{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, 
									stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Минимум</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_max">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(180, 180, 180);
	color: white;
	border: 1px solid gray;
}
QLineEdit: pressed{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, 
									stop: 0 #FF7832, stop: 1 #FF9739);
}
</string>
    </property>
    <property name="text">
     <string>Максимум</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_avg">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(180, 180, 180);
	color: white;
	border: 1px solid gray;
}
QLineEdit: pressed{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, 
									stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Медиана</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>11</y>
      <width>521</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
	background-color: rgb(180, 180, 180);
	color: white;
	border: 1px solid gray;
}
QTableWidget: pressed{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, 
									stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_region">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(60, 60, 60);
	color: white;
	border: 1px solid gray;
}
QLineEdit: pressed{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, 
									stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Название региона</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_path">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(60, 60, 60);
	color: white;
	border: 1px solid gray;
}
QLineEdit: pressed{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, 
									stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Путь к файлу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open_file">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 151, 57);
	color: white;
	border: 1px solid gray;
}
QPushButton: pressed{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, 
									stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Column">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(60, 60, 60);
	color: white;
	border: 1px solid gray;
}
QLineEdit: pressed{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, 
									stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Номер столбца</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Column">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 151, 57);
	color: white;
	border: 1px solid gray;
}
QPushButton: pressed{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, 
						stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Рассчет</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
